### ISSUE TITLE
Add support for `pass_filenames`

### ISSUE THREAD

**Issue by skycaptain** [pre-commit/pre-commit-mirror-maker#82]
I'm trying to setup a mirror for a Python script that does not expect any other arguments, including filenames. Currently `pre-commit-mirror` does not support this command and also removes it from `.pre-commit-hooks.yaml` when added before and the command is run again. This inhibits from using `pre-commit-mirror` e.g. in a Github Action to keep the mirror up-to-date.

**Comment by asottile** [comment]
`pass_filenames` is generally a hack and not recommended for general use -- say more about your usecase? what tool?  their tool should probably be improved to properly support filenames

**Comment by skycaptain** [comment]
@asottile Good to know. I'm trying to use `poetry check` from [poetry](https://github.com/python-poetry/poetry) which checks `pyproject.toml`. I'm aware of https://gitlab.com/smop/pre-commit-hooks, which is also listed at https://pre-commit.com/hooks.html, however their implementation does require poetry to be installed on the system externally as they are using `language: system`. One could argue that developers who work on projects using poetry have it installed on their system anyway, or I could add `additional_dependencies` to the downstream project, but that feels awkward rather than a clean solution.

**Comment by asottile** [comment]
yeah seems like `poetry check` could be improved to take (optional) filenames: https://github.com/python-poetry/poetry/blob/494dec279daeb3b52348a793598eb4861163403e/poetry/console/commands/check.py

this would also better allow a monorepo setup where you had something like:

```
proj1/
    pyproject.toml
proj2/
    pyproject.toml
```
